client/src/components/ProjectSearchResult.js

```diff
@@ -13,9 +13,8 @@ import { ProjectExternalAccessionsDetail } from 'components/ProjectExternalAcces
 
 export const ProjectSearchResult = ({ project }) => {
   const { responsive } = useResponsive()
-  const { isMetadataOnlyAvailable, metadataComputedFile } = useMetadataOnly(
-    project.computed_files
-  )
+  const { isMetadataOnlyAvailable, metadataComputedFile } =
+    useMetadataOnly(project)
```

Comment:
`useMetadataOnly` now takes entire data structure instead of just `computed_files`, so the passed argument has been updated. The same change applies to `ProjectSamplesTable`.